plugins/jobs/web_client/templates/JobList.pug

```diff
@@ -34,5 +34,3 @@ table.g-jobs-list-table
           td= job.get('type')
         if columns & columnEnum.COLUMN_STATUS
           td.g-job-status-cell= JobStatus.text(job.get('status'))
-
-.g-job-pagination
```

Comment:
Let's move this file back to its original name in case any downstreams were importing it statically.